ajax/libs/rivescript/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rivescript",
+  "filename": "rivescript.min.js",
+  "version": "1.17.1",
+  "description": "RiveScript is a scripting language for chatterbots, making it easy to write trigger/response pairs for building up a bot's intelligence.",
+  "keywords": [
+    "bot",
+    "chatbot",
+    "chatterbot",
+    "aiml",
+    "artificial intelligence",
+    "chatscript",
+    "buddyscript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aichaos/rivescript-js.git"
+  },
+  "homepage": "http://www.rivescript.com/",
```

Comment:
@pvnr0082t I've appended the `s` on the `http`. Please review this PR again. Thank you